data-extraction/recipes.json

```diff
@@ -0,0 +1,778 @@
+[
+    {
+        "id": 1,
+        "spoonacular-id": 333606,
+        "title": "Halibut with Lemon, Capers, and Croutons",
+        "image": "https://spoonacular.com/recipeImages/halibut-with-lemon-capers-and-croutons-333606.jpeg",
+        "ingredients": [
+            {
+                "amount": 1.0,
+                "id": 1001,
```

Comment:
In the recipe ingredients, is this the spoonacular id for the ingredients? Should it be our ingredient id instead?